index.js

```diff
@@ -37,8 +37,8 @@ function createStaleCacher(cache, locks, cacheStats, loader) {
     return function(id, callback) {
         // Stringify objects.
         var key = JSON.stringify(id);
-
-        // Instance is in LRU cache.
+        (!loader.name) ? lockedFunction = 'tlMapbox' : lockedFunction = loader.name;
```

Comment:
Does this need a `let lockedFunction;` at the top @riastrad ?

Reply:
It absolutely does. Good catch, @springmeyer! 🙇